man/covidcast_epidata.Rd

```diff
@@ -27,7 +27,14 @@ an object containing fields for every signal:
 
 \if{html}{\out{<div class="sourceCode r">}}\preformatted{epidata <- covidcast_epidata()
 epidata$signals
-#> # A tibble: 451 x 3
+#> Warning: Automatic coercion from logical to character was deprecated in purrr 1.0.0.
+#> i Please use an explicit call to `as.character()` within `map_chr()` instead.
+#> i The deprecated feature was likely used in the epidatr package.
+#>   Please report the issue at <https://github.com/cmu-delphi/epidatr/issues>.
+#> This warning is displayed once every 8 hours.
+#> Call `lifecycle::last_lifecycle_warnings()` to see where this warning was
+#> generated.
```

Comment:
you probbly dont want this in there, right?

Reply:
Nice catch. Somehow this wasn't happening at all when I called that code directly from the command line.

Reply:
oh apparently the bit @dshemetov changed ffixed that, thanks!